proto/redpanda/api/dataplane/v1/pipeline.proto

```diff
@@ -47,6 +47,11 @@ message Pipeline {
 
   Status status = 8 [(google.api.field_behavior) = OUTPUT_ONLY];
 
+  // External address. It is specific to the redpanda connect config and
+  // plugin, what address is returned, it could be
+  // http:// or wss:// or something else.
+  string external_address = 10;
```

Comment:
Naming is hard and I don't have strong opinion... but I wonder if "url" is better than "address"? Ex `"external_url"`? For most PaaS, it's some variation of that. Ex, `"live_url"`, `"web_url"`, or just `"url"`.